src/Makefile

```diff
@@ -24,6 +25,11 @@ schedule_professor:
 professors:
 	scrapy crawl professors
 
+info:
+	@echo $(shell pwd)
+	@echo $(shell ls)
```

Comment:
Is this needed? or it was a left over for debugging?

Reply:
Oopsie 😆